client/overview/index.js

```diff
@@ -181,6 +182,17 @@ const OverviewPage = () => {
 										/>
 									</ErrorBoundary>
 								</Card>
+								{
+									/* Show Payment Activity widget only when feature flag is set. To be removed before go live */
+									wcpaySettings.featureFlags
```

Comment:
This can be moved out of JSX block and saved in a constant.

```suggestion
			const { isPaymentOverviewWidgetEnabled ) = wcpaySettings.featureFlags
```

Reply:
Thanks for catching this! Certainly helps the code get more tidy. I have refactored this and a few other places where I thought destructuring could help. I hope the approach is fine: 

https://github.com/Automattic/woocommerce-payments/pull/8412/commits/992025a026825c50f057e5c4ad4f8fa24230ef48

https://github.com/Automattic/woocommerce-payments/pull/8412/commits/aba3603a233de0ec65ba0a1398f06efe6926ec3f

https://github.com/Automattic/woocommerce-payments/pull/8412/commits/5f9e3df463d3eb48c02f5a2540623c611668e2a7